backend/schema.sql

```diff
@@ -0,0 +1,64 @@
+/** Database schemas
+*   Entries of the database schema.
+*   Users table is the collection of all available users of our app.
+*   Events is the collection of all events during our Fellowship.
+*   Points is the record of all the points earned for fellows.
+*
+*/
+
+/** Roles Enum
+*   Use an enum to prevent using varchar for user roles
+*
+*/
+
+CREATE TYPE roles AS ENUM ('admin', 'fellow');
+
+/** Users
+*   
+*
+*/
+
+CREATE TABLE Users (
+    UserID INTEGER,
+    DisplayName VARCHAR(255) NOT NULL,
+    Role roles NOT NULL,
+    Pod INTEGER NOT NULL,
```

Comment:
Would this be VARCHAR? I'm thinking the Pod number would be stored, so something like '2.0.2' which idk if that would work with INTEGER

Reply:
I think you are right that it has to be a var char, overlooked by me!

Reply:
PR was resolved by implementing the same data model in models.py